cvat/apps/analytics_report/report/create.py

```diff
@@ -68,18 +68,18 @@ def _get_queue(self):
 
     def _make_queue_job_prefix(self, obj) -> str:
         if isinstance(obj, Task):
-            return f"{self._QUEUE_JOB_PREFIX_TASK}{obj.id}-"
+            return f"{self._QUEUE_JOB_PREFIX_TASK}{obj.id}"
         else:
-            return f"{self._QUEUE_JOB_PREFIX_PROJECT}{obj.id}-"
+            return f"{self._QUEUE_JOB_PREFIX_PROJECT}{obj.id}"
 
     def _make_custom_analytics_check_job_id(self) -> str:
         return uuid4().hex
 
-    def _make_initial_queue_job_id(self, obj) -> str:
-        return f"{self._make_queue_job_prefix(obj)}initial"
+    def _make_queue_job_id(self, obj, start_time: timezone.datetime) -> str:
+        return f"{self._make_queue_job_prefix(obj)}-{start_time.timestamp()}"
 
-    def _make_regular_queue_job_id(self, obj, start_time: timezone.datetime) -> str:
-        return f"{self._make_queue_job_prefix(obj)}{start_time.timestamp()}"
+    def _make_autoupdate_blocker_key(self, obj) -> str:
+        return f"cvat:analytics:autoupdate-blocker:{self._make_queue_job_prefix(obj)}"
```

Comment:
I dont think we should put something named `prefix` to the end of the key, its better to be in the beggining

Reply:
Yeah, that's fair enough. I think the format of the key itself is fine, but I can rename the function to something more semantically fitting.

Reply:
Renamed.